mm2src/coins/Cargo.toml

```diff
@@ -16,7 +16,7 @@ enable-solana = [
 	"dep:spl-associated-token-account"
 ]
 enable-nft-integration = []
-default = []
+default = ["enable-nft-integration"]
```

Comment:
fyi: If we want NFT integration enabled on releases, it's better to remove this flag completely. If this is only for providing build output from upload server, it can stay until the merge.

Reply:
I removed nft flag, bcz we can use proxy in releases